test/integration/taxon_browsing_test.rb

```diff
@@ -91,6 +91,17 @@ class TaxonBrowsingTest < ActionDispatch::IntegrationTest
     and_i_cannot_see_an_email_signup_link
   end
 
+  it 'sets up a non-live taxon with no grandchildren' do
+    given_there_is_a_taxon_without_grandchildren
+    and_the_taxon_without_grandchildren_is_not_live
+    and_there_are_popular_items_for_the_taxon
+    when_i_visit_the_taxon_page
+    then_i_can_see_there_is_a_page_title
+    and_i_can_see_the_general_information_section_in_the_accordion
+    and_i_can_see_links_to_the_child_taxons_in_an_accordion
+    and_i_cannot_see_an_email_signup_link
```

Comment:
I think we also need to test that the email subscription link does appear in each of the sections in the accordion (ie child taxons).... 

You could do a test like `and_all_sections_do_not_have_an_email_signup_link`, which is the opposite to this test: https://github.com/alphagov/collections/blob/e36a7752674116c853b83c45d1798b9619019396/test/integration/taxon_browsing_test.rb#L424

Reply:
`and_i_cannot_see_an_email_signup_link` checks the whole page, so the test should already fail if sections in the accordion have email signup links.

Reply:
Oh of course!! Nice!